# Optional 

robot_name: "Jackal" #string

# Key in '0' for 2D Lidar
# Key in '1' for 3D Lidar

lidar_type: 0 # int

# if choosing 3D boundary box, set height
min_height: 0.1

# Boundary Box 2D Type
# '0' for triangle
# '1' for circle
# '2' for square
# '3' for ellipse
# '4' for pentagon

# '5' for sphere
# '6' for cuboid
# '7' for ellipsoid

boundary_box: 4

# Boundary Box Shape Specific Params
# Only fill in for shape required
# units in meters

# height is along the y-axis
triangle:
  height: 4
  breath: 3

# Distance from origin left x/right x, top y/bottom y
circle:
  radius: 1.5

square:
  sq_side: 3.0

# vertical ellipse, ie the longer side is along the y-axis
ellipse:
  major_axis: 2.0
  minor_axis: 1.0

pentagon:
  len_from_mid_to_vertice: 1.5
  
sphere:
  radius: 1.0

cuboid:
  side_length: 3.0

ellipsoid:
  major_axis: 1.5
  semi_major_axis: 0.7
  minor_axis: 0.5

